plugin-volume/volumepopup.cpp

```diff
@@ -42,6 +42,16 @@
 #include <QDebug>
 #include <QWheelEvent>
 
+// some "modern" icon themes have a specific panel icon
+static QString panelIcon(const QString &iconName)
```

Comment:
This is not necessary and using the "*-panel" icons is enough, because  XDG search algorithm uses the "dash" to separate levels of specificity in icon names..

Or am I missing something?